dapr/clients/__init__.py

```diff
@@ -7,7 +7,7 @@
 
 from dapr.clients.base import DaprActorClientBase
 from dapr.clients.exceptions import DaprInternalError, ERROR_CODE_UNKNOWN
-from dapr.clients.grpc.client import DaprClient
+from dapr.clients.generic import DaprClient
```

Comment:
`generic` is an overloaded term here.  Can you name this package something else? Maybe `main`?

Reply:
Yeah, can do.